src/controller/projections.rs

```diff
@@ -90,7 +90,9 @@ pub fn update_orthographic(mut cameras: Query<(&mut EditorCam, &mut Projection,
         };
 
         let anchor_dist = editor_cam.last_anchor_depth().abs() as f32;
-        let target_dist = (editor_cam.orthographic.scale_to_near_clip * orthographic.scale).clamp(
+        let target_dist = (editor_cam.orthographic.scale_to_near_clip
+            * (orthographic.area.width() * orthographic.area.height()))
```

Comment:
This is probably wrong, but I'm not sure what to put there.

Reply:
I think scale has been replaced with setting it here: https://dev-docs.bevyengine.org/bevy/render/camera/enum.ScalingMode.html

Reply:
It defaults to `ScalingMode::WindowSize(1.0)`, and now if you were using scale of 2.0, you would now do `ScalingMode::WindowSize(2.0)`. You will need to match on the scaling enum and extract the multiplier. I'm not sure if we can support anything other than windowsize, or if there is a better way to do this.